make/libraries

```diff
@@ -131,7 +137,18 @@ ifeq (Linux, $(OS))
   SHELL = /usr/bin/env bash
 endif
 
+# If brackets or spaces are found in MAKE on Windows
+# we error, as those characters cause issues when building.
+ifeq (Windows_NT, $(OS))
+MAKE_ESCAPED := ${subst ),,${subst (,,$(lastword $(MAKE))}}
+else
+MAKE_ESCAPED := $(MAKE)
+endif
+
 $(TBB_BIN)/tbb-make-check:
+ifneq ($(MAKE),$(MAKE_ESCAPED))
+$(error '$nError:$n$nThe toolchain is installed in a path with spaces or brackets.$nPlease reinstall the toolchain.$n$n')
```

Comment:
Add rtools somewhere -- this is only happening in Windows right and the only toolchain people are using is rtools?

Reply:
Good call. And yes to both questions.

Reply:
New message is: 

The RTools toolchain is installed in a path with spaces or brackets.
Please reinstall the toolchain.